frontend/src/components/HomePage/index.jsx

```diff
@@ -24,6 +25,38 @@ const addCampaignBillsToSearchObj = (campaignMappings, searchObj) => {
   return { ...searchObj, basePrintNo: Array.from(billIds) };
 };
 
+/** Get direct sponsors for all bills. Does _not_ include cosponsors
+ *
+ * @param {Array} bills - Array of bill objects from legislative API
+ * @returns {Array} - Array of sponsor objects in format for Dropdown: {displayName: string, value: string (id)}
+ */
+const getSponsorList = (bills) => {
+  // Use set to prevent duplicates
+  const sponsorIds = new Set();
+
+  // Create list of sponsor objects
+  let sponsorList = bills
+    .map((bill) => {
+      const sponsorObj = getSponsorMembers(bill);
+      return {
+        displayName: sponsorObj?.sponsor?.fullName,
+        value: sponsorObj?.sponsor?.memberId,
+      };
+    })
+    .filter((x) => !!x.displayName && !!x.value);
+
+  // Filter out dupes
+  sponsorList = sponsorList.filter((sponsorOption) => {
+    if (sponsorIds.has(sponsorOption.value)) {
+      return false;
+    }
+    sponsorIds.add(sponsorOption.value);
+    return true;
+  });
```

Comment:
i think it'd be helpful to sort these, maybe alphabetically:
```
    // sort alphabetically, from first name
    sponsorBills.filter(/*...*/).sort((a, b) => {
      if (a.displayName < b.displayName) {
        return -1;
      }
      if (a.displayName > b.displayName) {
        return 1;
      }
      return 0;
    });
```

Reply:
Makes sense 👍